libs/client/editor-lite/src/lib/editor-lite.component.html

```diff
@@ -62,15 +62,23 @@
         <div class="mx-0.5">|</div>
         <button
             class="shadow-none border-none px-2.5 py-1.5 hover:scale-100"
-            (click)="openInsertMedia('Insert Image')"
+            [ngClass]="{'active': imageMenuOpened}"
+            [matMenuTriggerFor]="insertImageMenu"
+            (menuOpened)="openImageMenu()"
+            (menuClosed)="openImageMenu()"
             [matTooltip]="'Insert Image'"
             [matTooltipClass]="'offprint-tooltip'"
         >
             <rmx-icon name="image-add-line" class="mr-0"></rmx-icon>
         </button>
         <button
             class="shadow-none border-none px-2.5 py-1.5 hover:scale-100"
-            (click)="openInsertMedia('Insert Media')"
+            [ngClass]="{'active': mediaMenuOpened}"
+            [matMenuTriggerFor]="insertMediaMenu"
+            (menuOpened)="openMediaMenu()"
+            (menuClosed)="openImageMenu()"
+            [matTooltip]="'Insert Media'"
```

Comment:
Maybe this should also say "Embed YouTube video"?

Reply:
see last comment